CMakeFilters.cmake

```diff
@@ -10,8 +10,8 @@
 # help@hdfgroup.org.
 #
 option (USE_LIBAEC_STATIC "Use static AEC library" OFF)
-option (ZLIB_USE_EXTERNAL "Use External Library Building for ZLIB" OFF)
-option (SZIP_USE_EXTERNAL "Use External Library Building for SZIP" OFF)
+option (ZLIB_USE_EXTERNAL "Use External Library Building for ZLIB else search" OFF)
```

Comment:
I feel like "else search" doesn't give much information and would just make this even more confusing for users, but I don't have any particular evidence to back that up.

Reply:
User sstill complained it was unclear and that is what happens search using find_package

Reply:
Right, I'm saying maybe that needs to be even more explicit, such as "else search system with find_package", even though I still think that isn't necessarily helpful for most people that just want to build their HDF5 with zlib support and don't know what find_package is.